component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/totp/TOTPAuthenticatorConstants.java

```diff
@@ -35,6 +35,7 @@ private TOTPAuthenticatorConstants() {
 	public static final String AUTHENTICATOR_NAME = "totp";
 	public static final String QR_CODE_CLAIM_URL = "http://wso2.org/claims/identity/qrcodeurl";
 	public static final String SECRET_KEY_CLAIM_URL = "http://wso2.org/claims/identity/secretkey";
+	public static final String TOTP_ENABLED_CLAIM_URI = "http://wso2.org/claims/identity/totpEnabled";
```

Comment:
Have you added this to the claim config file so that it'll be available by default in the product?
Is the migration from the previous versions captured?

Reply:
PR to add claim to the product 
-